README.md

```diff
@@ -1,12 +1,24 @@
 # integration-adaptor-lab-results
-Integration Adaptor to simplify processing of Pathology and Screening results
+Integration Adaptor to simplify processing of Pathology and Screening results.
 
 ## Configuration
 
 The adaptor reads its configuration from environment variables. The following sections describe the environment variables
  used to configure the adaptor. 
  
-Variables without a default value and not marked optional *MUST* be defined for the adaptor to run.
+Variables without a default value and not marked optional are *MUST* be defined for the adaptor to run.
```

Comment:
I'd revert this change. The `are` doesn't read right

Reply:
done